freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -251,12 +258,24 @@ def handle(self, event):
                 # to rebuild list
                 to_rebuild_digests.add(digest)
 
-        # Skip that event because we can't proceed with processing it.
-        # TODO
-        # Now when we have bundle images' nvrs we can procceed with rebuilding it
-        msg = f"Skipping the rebuild of {len(to_rebuild_digests)} bundle images " \
-              "due to being blocked on further implementation for now."
-        db_event.transition(EventState.SKIPPED, msg)
+        builds = self._prepare_builds(db_event, bundles_by_digest,
+                                      to_rebuild_digests)
+
+        # Reset context to db_event.
+        self.set_context(db_event)
+
+        if not builds:
```

Comment:
Could you move this check before calling `self._prepare_builds` and have it check to see if `to_rebuild_digests` is empty instead of `builds`? Having it this way implies that `_prepare_builds` might skip entries that are present in `to_rebuild_digests`.

Reply:
Sure